samples/tables/lib/dual-tables.js

```diff
@@ -0,0 +1,100 @@
+import {MathML} from "../../../mathjax3/input/mathml.js";
+import {CHTML} from "../../../mathjax3/output/chtml.js";
+import {SVG} from "../../../mathjax3/output/svg.js";
+import {HTMLMathItem} from "../../../mathjax3/handlers/html/HTMLMathItem.js";
+import {HTMLDocument} from "../../../mathjax3/handlers/html/HTMLDocument.js";
+import {handleRetriesFor} from "../../../mathjax3/util/Retries.js";
+import {browserAdaptor} from "../../../mathjax3/adaptors/browserAdaptor.js";
+
+let mml = new MathML({forceReparse: true});
+let chtml = new CHTML({fontURL: '../../mathjax2/css/'});
+let svg = new SVG();
+
+let docs = {
+  CHTML: new HTMLDocument(document, browserAdaptor(), {InputJax: mml, OutputJax: chtml}),
+  SVG:   new HTMLDocument(document, browserAdaptor(), {InputJax: mml, OutputJax: svg})
+};
+
+const samples = [
+  '<mjx-samples>',
+  '<mjx-description>',
+  '',
+  '</mjx-description>',
+  '<mjx-tables>',
+  '<mjx-chtml-table>',
+  '',
+  '</mjx-chtml-table>',
+  '<mjx-svg-table>',
+  '',
+  '</mjx-svg-table>',
+  '</mjx-tables>',
+  '</mjx-samples>'
+];
+
+const div = document.createElement('div');
+const template = document.getElementsByTagName('mjx-template')[0];
+
+function  CreateMathML() {
+    const mathml = template.innerHTML;
+    template.innerHTML = mathml.replace(/\&/g, '&amp;')
+                               .replace(/</g, '&lt;')
+                               .replace(/>/g, '&gt;')
+                               .replace(/\n/g, '<br/>')
+                               .replace(/\s/g, '\u00A0');
```

Comment:
Any chance this might ever change or grow? Then it might be easier to put it into a pair list that's processed.

Reply:
I don't think so.  This is to format the original HTML source from the template so that it is displayed as code in the browser.  These are the things that need to be changed to do that (HTML specials, newlines and spaces).   I suppose the spaces and newlines could be handed by setting the CSS for white-space, but otherwise, this should not need to be changed.